src/commands/moderation/timeout.ts

```diff
@@ -149,7 +149,7 @@ const failedTimeoutEmbed = (user: User) => (makeEmbed({
 
 export const timeout: CommandDefinition = {
     name: 'timeout',
-    requiredPermissions: ['BanMembers'],
+    requirements: { permissions: ['BanMembers'] },
```

Comment:
Not a comment on the PR, but @benw202 : Does this make sense, should we take the opportunity to change this to ADMIN & MODERATOR roles?